lib/Db/PublicKeyCredentialEntityMapper.php

```diff
@@ -93,6 +93,23 @@ public function __construct(IDBConnection $db) {
 	 */
 	protected $active;
 
+	public function findById(int $id): ?PublicKeyCredentialEntity {
+		$qb = $this->db->getQueryBuilder();
+
+		$qb->select('*')
+			->from($this->getTableName())
+			->where($qb->expr()->eq(
+				'id',
+				$qb->createNamedParameter($id, IQueryBuilder::PARAM_INT),
+				IQueryBuilder::PARAM_INT,
+			));
+		try {
+			return $this->findEntity($qb);
+		} catch (\Exception) {
```

Comment:
a bit too generic for my taste :speak_no_evil:

Reply:
better: catch not found exception only

Reply:
Done